### DISCUSSION TITLE
Does MLFlow have API's for generation of Model cards?

### DISCUSSION THREAD

**Discussion by bhargav-minto** [mlflow/mlflow#7969] [Q&A] [answered]
hello everyone, 
is there a way to generate a model card with the information stored using MLflow Tracking? 

Thanks,
Bhargav.

**Comment by JHibbard** [comment, marked as answer]
Hi Bhargav,

I don't believe there's a specific `model card` API but you could write a helper function to pull the information from the tracking server to make your own model card. This would also be a great contribution to the project if you're interested in contributing a feature.

Hope that helps!
Jim
